src/aria2.cpp

```diff
@@ -199,5 +219,25 @@ void Aria2::shutdown()
   doRequest(methodCall);
 }
 
+void Aria2::forcePause(const std::string& gid)
+{
+    MethodCall methodCall("aria2.forcePause", m_secret);
```

Comment:
I would prefer to use the "pause" method instead of "forcePause" (same for "remove"/"forceRemove").
The normal behavior should be "pause" to allow aria2 to correctly close the connection. We should use "force*" only if really necessary.